eclass/distutils-r1.eclass

```diff
@@ -1828,6 +1828,14 @@ distutils-r1_run_phase() {
 		# Rust extensions are incompatible with C/C++ LTO compiler
 		# see e.g. https://bugs.gentoo.org/910220
 		if has cargo ${INHERITED}; then
+			# see all-flag-vars in flag-o-matic.eclass
+			local -x ADAFLAGS="${ADAFLAGS}"
+			local -x CFLAGS="${CFLAGS}"
```

Comment:
I wonder if we should actually iterate all-flag-vars here:

```bash
for v in $(all-flag-vars); do
    local -x ${v}="${!v}"
done
```

Reply:
I was wondering that. Thanks!